geonode/assets/local.py

```diff
@@ -62,6 +62,18 @@ def create(self, title, description, type, owner, files=None, clone_files=False,
 
         if clone_files:
             files = self._copy_data(files)
+        elif isinstance(files[0], str):
+            pass
+        else:
+            # if files are in-memory, need to handle them properly
+            new_path = self._create_asset_dir()
+            relative_dir = os.path.relpath(new_path, os.path.dirname(settings.ASSETS_ROOT))
+            processed_files = []
+            for f in files:
+                save_path = os.path.join(relative_dir, f.name)
+                self.get_storage_manager(None).save(save_path, f)
```

Comment:
@etj do u think is better to use the `save` here or the `clone_remote_files`? The `save` method is not passing by the [DataRetriever](https://github.com/GeoNode/geonode/blob/423029f503c96b39bee93e6982d59dd546ed7b68/geonode/storage/data_retriever.py#L146) so the permissions, unzipping and [transfer in chunk](https://github.com/GeoNode/geonode/blob/423029f503c96b39bee93e6982d59dd546ed7b68/geonode/storage/data_retriever.py#L86) is not used